raspberry-pi/4/pkgs-overlays.nix

```
@@ -3,6 +3,7 @@ let
   cfg = config.hardware.raspberry-pi."4".apply-overlays-dtmerge;
   dt_ao_overlay = _final: prev: {
     deviceTree.applyOverlays = prev.callPackage ./apply-overlays-dtmerge.nix { };
+    deviceTree.compileDTS = prev.callPackage ./compile-overlays-dtmerge.nix { };
```

Adding `compileDTS` is unnecessary. Instead, override `applyOverlays` properly:

```suggestion
    deviceTree = prev.deviceTree // {
      applyOverlays = prev.callPackage ./apply-overlays-dtmerge.nix { };
    };
```